package.json

```diff
@@ -62,6 +62,7 @@
     "@types/styled-components": "^5.1.3",
     "react-autosize-textarea": "^7.1.0",
     "react-copy-to-clipboard": "^5.0.2",
+    "react-in-out-textarea": "^0.0.3",
```

Comment:
I think you do not have to include that here. Why was this added?

Reply:
Im not sure why my system added that. Ill fix it!

Reply:
Hmm I am not sure why yarn added that. I will fix it.